wp-admin/js/widgets/media-widgets.js

```diff
@@ -134,13 +160,13 @@ wp.mediaWidgets = ( function( $ ) {
 						});
 
 						embedLinkView.dfd.done( function( response ) {
-							embedLinkView.renderoEmbed( {
-								data: {
-									body: response.html
-								}
-							});
+							if ( embedLinkView.controller.options.mimeType !== response.type ) {
+								embedLinkView.renderFail();
```

Comment:
This will now `renderFail` on any service that doesn't have embed discovery - for certain any now that are not supported by noembed... which maybe is okay - but something we should consider